### ISSUE TITLE
The nodegit commit class does not store username, it only stores full name and email.

### ISSUE THREAD

**Issue by shoebilldev** [kblincoe/VisualGit_SE701_3#122]
The nodegit commit class does not store username. It would be good to have a way to find the username of the committer, instead of just the email or full name.

Below is the documentation for nodegit's author and committer api and the information they hold about committers.

http://www.nodegit.org/api/commit/#author
http://www.nodegit.org/api/commit/#committer

You could search github's API for the full name or email but people share full names, not everyone has emails public and not everyone only has one account with each email.

This issue blocks #27 

@softeng-701

**Comment by softeng-701** [comment]
@tagumi Approved, does this additional issue mean you are happy with the size of #27 ?

**Comment by shoebilldev** [comment]
@softeng-701 yes, with the approval of this issue I think the size of #27 as medium is appropriate
